### ISSUE TITLE
v0.05 not show like code style

### ISSUE THREAD

**Issue by gh16683170** [valentine195/links-in-code-block#5]
v0.0.4 can show code block like code style , but v0.05 show code block like normal markdown, big gap bettween two lines, and can not show the blanks in source code. 
![image](https://user-images.githubusercontent.com/16650849/112860036-0672d980-90e6-11eb-9c02-16e246ebddb8.png)

**Comment by gh16683170** [comment]
the normal markdown style can not show the blanks and other style, so we need the code block. thanks.

**Comment by valentine195** [comment]
Closed in 0.1.0

0.1.0 also added the ability to tell the plugin what language (if any) you're writing, and it will use Obsidian's code highlighter:

````markdown
```links|clike
#include

int main(int argc, char** argv)
{
/* add your codes here */
	printf("hello world ! \n");
	return 0;
	[[ZZZZZ Plugins/Untitled]]
	[[not created]]
}
```
````

![image](https://user-images.githubusercontent.com/38669521/112885079-4ada8700-909e-11eb-9e8a-eb663fd3d3f0.png)
